AzerothAuctionAssassin.py

```
@@ -5,7 +5,7 @@
 import sys
 from datetime import datetime
 
-AAA_VERSION = "1.2.6"
+AAA_VERSION = open("version.txt", "r").read().strip()
```

Ruff (0.8.0)

Use a context manager for opening files

(SIM115)